<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="searchFindLine"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="searchFindButton">
     <property name="text">
      <string>Find next</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="searchFindAllButton">
     <property name="text">
      <string>Find all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace with</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="searchReplaceLine"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="searchReplaceButton">
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="searchReplaceAllButton">
     <property name="text">
      <string>Replace all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="searchWholeBox">
     <property name="text">
      <string>Whole words</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="searchCaseBox">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QCheckBox" name="searchWrapBox">
     <property name="text">
      <string>Wrap search</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchFindLine</tabstop>
  <tabstop>searchReplaceLine</tabstop>
  <tabstop>searchFindButton</tabstop>
  <tabstop>searchReplaceButton</tabstop>
  <tabstop>searchReplaceAllButton</tabstop>
  <tabstop>searchCaseBox</tabstop>
  <tabstop>searchWholeBox</tabstop>
  <tabstop>searchWrapBox</tabstop>
  <tabstop>searchFindAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>searchFindLine</sender>
   <signal>returnPressed()</signal>
   <receiver>searchFindButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchReplaceLine</sender>
   <signal>returnPressed()</signal>
   <receiver>searchReplaceButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
